geometry/renderPng.js

```diff
@@ -1,4 +1,5 @@
-import UPNG from 'upng-js';
+// import UPNG from 'upng-js';
+import { encode } from 'fast-png';
 import gl from 'gl';
 import { staticDisplay } from './threejs.js';
```

Comment:
Interesting, so is the demo actually using 3js, but it's just a static version right now?

Reply:
Yeah, since that works in the cli and for tests.

I'll add support for the orbital view into the web version in a bit.

(Which is why I separated the cli and web view ops)